android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -127,6 +134,9 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
                 // by default we hide.
                 options.setAttachThreads(rnOptions.getBoolean("attachThreads"));
             }
+            if (rnOptions.hasKey("attachScreenshot")) {
+                options.setAttachScreenshot(rnOptions.getBoolean("attachScreenshot"));
+            }
```

Comment:
H: This alone won't work for Native events, See https://github.com/getsentry/sentry-react-native/issues/2362#issuecomment-1193763553
Maybe we should expose a `ScreenshotEventProcessor#setCurrentActivity(activity)` so the RN SDK can overwrite this value when initing the Android SDK.